packages/video_player/video_player/test/video_player_initialization_test.dart

```diff
@@ -0,0 +1,22 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/widgets.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:video_player/video_player.dart';
+
+import 'video_player_test.dart' show FakeVideoPlayerPlatform;
+
+void main() {
+  test('plugin initialized', () async {
```

Comment:
nit: add a comment explaining that this test should be the only test in this file as it needs to run first

Reply:
Done